README.md

```diff
@@ -103,7 +117,7 @@ Remember to remove the lines from `simulator/internal/internal.go` before
 committing changes.
 ```
 // #cgo CFLAGS: -I $OPENSSL_PATH/include
-// #cgo LDFLAGS: -L$OPENSSL_PATH/lib
+// #cgo LDFLAGS: -L $OPENSSL_PATH/lib
```

Comment:
maybe tell the user to replace existing lines prefixed with "#cgo darwin CFLAGS" now that we've added the lines in for Github actions

```
// #cgo darwin CFLAGS: -I $OPENSSL_PATH/include
// #cgo darwin LDFLAGS: -L $OPENSSL_PATH/lib
```

Reply:
Updated the `README`